com.unity.shadergraph/Editor/Data/Nodes/Utility/CustomFunctionNode.cs

```diff
@@ -422,9 +411,9 @@ public override void ValidateNode()
             base.ValidateNode();
         }
 
-        public bool Reload(HashSet<string> changedFileDependencies)
+        public bool Reload(HashSet<string> changedFileDependencyGUIDs)
```

Comment:
this is just changing the name of the variable to make it clear these are GUIDs not paths